### ISSUE TITLE
Only Small Object Branch Can Detect The Target

### ISSUE THREAD

**Issue by michaelowenliu** [YunYang1994/tensorflow-yolov3#296]
1) I just found only "pred_sbbox" branch outputs the valid results. 
2) the loss would become NaN after around 20 epochs while training from scratch.

**Comment by fengxiuyaun** [comment]
me too!!!!!

**Comment by michaelowenliu** [comment]
> me too!!!!!

just solved the first point. Divide anchor by strides, i.e. [8,16,32], like the format in baseline_anchors.txt, but not like coco_anchors.txt.
